libgraph/include/katana/GraphTopology.h

```diff
@@ -1804,16 +1814,23 @@ class KATANA_EXPORT PGViewCache {
         pg, node_types, edge_types, *this);
   }
 
+  // Avoids a copy of the default topology.
+  const GraphTopology& GetDefaultTopologyRef() const noexcept;
+
+  void DropAllTopologies() noexcept;
```

Comment:
We should never be dropping all topologies. Haven't looked at the impl yet but I would prefer a name like `DropUnusedTopologies` which reflects this. One topology will always be in use by the PG, others may also be in use by views

Reply:
This was added in support of `katana::PropertyGraphRetractor::DropTopologies()` which does drop all topologies. So I didn't change any semantics here.

Reply:
Okay. I think this is the wrong behavior in that case though. We should not be constructing an empty default topology. We should either leave no topologies in the cache and force loading from storage or maintain a single valid topology.

Reply:
@witchel @tylershunt FYI